torch/utils/tensorboard/writer.py

```diff
@@ -292,7 +292,8 @@ def add_scalar(self, tag, scalar_value, global_step=None, walltime=None):
         self._get_file_writer().add_summary(
             scalar(tag, scalar_value), global_step, walltime)
 
-    def add_scalars(self, main_tag, tag_scalar_dict, global_step=None, walltime=None):
+    def add_scalars(self, main_tag, tag_scalar_dict, global_step=None,
+                    walltime=None):
```

Comment:
Nit - can you move the `walltime=None` back to the previous line for no changes here?